holidays/countries/spain.py

```diff
@@ -210,31 +241,29 @@ def _populate(self, year):
             self._is_observed(date(year, SEP, 17), "Día de Melilla")
             if year == 2022:
                 self._is_observed(
-                    _islamic_to_gre(year, 10, 1)[0] + rd(days=+1),
-                    "Aid Al-Fitr",
+                    _islamic_to_gre(year, 10, 1)[0] + rd(days=+1), "Eid Fitr"
                 )
                 self._is_observed(
                     _islamic_to_gre(year, 12, 10)[0] + rd(days=+2),
-                    "Aid Al-Adha",
-                )
-            else:
-                self._is_observed(
-                    _islamic_to_gre(year, 10, 1)[0], "Aid Al-Fitr"
+                    "Eid al-Adha",
                 )
+            elif year == 2023:
+                self._is_observed(_islamic_to_gre(year, 10, 1)[0], "Eid Fitr")
```

Comment:
Optional: Probably you know these holiday names better. My approach to naming  would be either "Eid Fitr, Aid Al Adha" (as per the reference you've provided) or "Eid al-Fitr, Eid al-Adha" (as per the Wikipedia).

Reply:
Done